weave/trace_server/update_cost_table.py

```diff
@@ -0,0 +1,60 @@
+# script to load data into llm_token_prices table in ClickHouse
```

Comment:
I think this routine might need some modifications to be production ready - perhaps it should be excluded from this PR?